packages/salesforcedx-vscode-soql/package.json

```diff
@@ -36,9 +36,9 @@
     "@salesforce/apex-tmlanguage": "1.5.0",
     "@salesforce/core": "^2.15.2",
     "@salesforce/salesforcedx-utils-vscode": "50.15.0",
-    "@salesforce/soql-builder-ui": "0.0.29",
+    "@salesforce/soql-builder-ui": "0.0.30",
```

Comment:
these dependencies haven't been published yet.  but will when the feature is ready for release.